pkg/array/array.go

```diff
@@ -254,30 +254,55 @@ func GetPowerStoreArrays(fs fs.Interface, filePath string) (map[string]*PowerSto
 	return arrayMap, mapper, defaultArray, nil
 }
 
-// ParseVolumeID parses volume id in from CO (Kubernetes) and tries to understand what in it are PowerStore volume id, and what is ip, protocol.
-// "/" is used as a delimiter.
+// ParseVolumeID parses a volume id from the CO (Kubernetes) and tries to extract the PowerStore volume id, Global ID, and protocol.
 //
 // Example:
 //
-//	ParseVolumeID("1cd254s/192.168.0.1/scsi") will return
-//		id = "1cd254s"
-//		ip = "192.168.0.1"
+//	ParseVolumeID("1cd254s/192.168.0.1/scsi") assuming 192.168.0.1 is the IP array PSabc0123def will return
+//		localVolumeID = "1cd254s"
+//		arrayID = "PSabc0123def"
 //		protocol = "scsi"
+//		e = nil
+//
+// Example:
+//
+//	ParseVolumeID("9f840c56-96e6-4de9-b5a3-27e7c20eaa77/PSabcdef0123/scsi:9f840c56-96e6-4de9-b5a3-27e7c20eaa77/PS0123abcdef") returns
```

Comment:
For any chance can this (volume ID) go beyond 253 char in any case?

Reply:
Luke may not be able to respond in time. What are the concerns?

Reply:
All components of the volume are known and handled by the `CreateVolume` k8s gRPC implementation in controller.go, so this is as long as it should ever be.